web/core/components/issues/issue-detail-widgets/relations/content.tsx

```diff
@@ -150,6 +166,21 @@ export const RelationsCollapsibleContent: FC<Props> = observer((props) => {
           }}
           data={issueCrudState?.delete?.issue as TIssue}
           onSubmit={async () => {
+            if (
+              issueCrudState.removeRelation.issue &&
+              issueCrudState.removeRelation.issue.project_id &&
+              issueCrudState.removeRelation.relationKey &&
+              issueCrudState.removeRelation.relationIssueId
+            ) {
+              await removeRelation(
+                workspaceSlug,
+                issueCrudState.removeRelation.issue.project_id,
+                issueCrudState.removeRelation.issueId as string,
```

Comment:
Do we really need the `as string` here? We're already checking the type in the if condition, so the type should be narrowed automatically and compatible after that check, right? Am I missing something?